### PR TITLE
don't exec if $EDITOR is empty, die with message

### PR DESCRIPTION
**Pull request by borisdaeppen**
EDITOR seems not to be set by default on my xubuntu.
So I always run into this error:

```
user@host:~/repos/open-this[master]$ ot Open::This
Use of uninitialized value in exec at /home/user/perl5/perlbrew/perls/perl-5.22.4/bin/ot line 43.
Can't exec "": Datei oder Verzeichnis nicht gefunden at /home/user/perl5/perlbrew/perls/perl-5.22.4/bin/ot line 43.
```
This error can the be fixed like so:
`user@host:~/repos/open-this[master]$ EDITOR=vim ot Open::This`
(or in `.bashrc` for sure)

With this patch it would die like this, which is a bit more kind:
```
user@host:~/repos/open-this[master]$ EDITOR='' perl script/ot Open::This
nothing in EDITOR at script/ot line 43.
```

I thought about making this check at the start of the script. But I thought, maybe `-b` should run, even if EDITOR is empty...

### CODE REVIEW

File: script/ot [hunk 1]
```diff
@@ -40,7 +40,7 @@ if ($browse) {
 
 my @editor_args = editor_args_from_parsed_text($parsed);
 
-exec $ENV{EDITOR}, @editor_args;
+$ENV{EDITOR} ? exec $ENV{EDITOR}, @editor_args : die('nothing in EDITOR');
```
Comment (oalders) [inline script/ot]: Could we change this to something like `die('Please ensure your $EDITOR env var has been set')`?  I think that would be even clearer.

**Comment by oalders** [comment]
Thanks very much for correcting my assumption.  👍

**Comment by borisdaeppen** [comment]
I made a new commit with the proposed error message

**Comment by oalders** [comment]
Thanks @borisdaeppen!

**Pull request merged**